docker-images/test-client/Dockerfile

```diff
@@ -7,6 +7,8 @@ ENV JAVA_OPTS "-DLOG_LEVEL=info"
 # copy scripts for starting Kafka
 COPY scripts $KAFKA_HOME
 COPY scripts/dynamic_resources.sh /bin/dynamic_resources.sh
+COPY kafka_client_alice_jass.conf $KAFKA_HOME/config
```

Comment:
@Frawless Do we need these files? I never used the test clients so I'm not sure how are they useful. I guess you are generating them on the fly?